src/features/bookReader/components/BookPDF.tsx

```
@@ -0,0 +1,193 @@
+      image?: { cloudinary?: { secure_url?: string } }
+    }
+
+function renderLexicalNode(node: LexicalNode): React.ReactNode {
+  if (!node) return null
+
+  switch (node.type) {
+    case 'root':
+      return node.children?.map((child, i) => <View key={i}>{renderLexicalNode(child)}</View>)
+
+    case 'paragraph':
+      const align = node.format === 'center' ? styles.center : {}
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)